src/introduction/README.md

```diff
@@ -16,40 +18,40 @@ and software stacks.
 ## Skills development exercises
 
 **Skills development exercises** are intended to take 1-2 hours each,
-and ask you to build and perform minor modifications to simple
-RISC-V and CHERI-RISC-V C/C++ programs. These exercises
-facilitate building skills such as compiling, executing,
-and debugging RISC-V and CHERI-RISC-V programs, as well as to build basic
-understanding of CHERI C/C++ properties. We highlight some key edge
-cases in CHERI, including the effects of bounds imprecision, subobject
-bounds, weaker temporal safety, and C type confusion.
+and ask you to build and perform minor modifications to simple C/C++
+programs. These exercises facilitate building skills such as
+compiling, executing, and debugging programs with both CHERI enabled
```

Comment:
Maybe "with and without CHERI extensions"?

Reply:
Reworked in the file.